components/Card.vue

```diff
@@ -197,6 +198,16 @@ export default Vue.extend({
       }
       return '';
     },
+    /**
+     * @returns {ImageType} Returns the relavent Image type based on the card size and item type
+     */
+    getImageType(): ImageType {
+      if (this.shape === 'thumb-card' && this.item.Type === 'Movie') {
+        return ImageType.Backdrop;
```

Comment:
What if the item has no backdrop? Maybe we should have a fallback, but I'm not sure what the behavior is for web currently.

Reply:
It looks like it uses the `thumb` image

Reply:
The cards don't really have a fallback, they're supposed to display the primary image. There are a few exceptions for shows and movies, which will sometimes show parent images.

- season cards will show series images
- episodes will show series backdrops

There are also a few backups that don't make sense at all since the aspect ratios are completely different, like using backdrops as a replacement for poster images. We shouldn't include those ones if you add backups.

I could also see some new backups getting implemented, like album artist replacing artist images, or album artist replacing album images, but I don't think backups are that important in general.

Reply:
The current web shows the `thumb` card. I think it would be better not to have a fall-back and just show the relevant icon instead.

Reply:
Hm, I think potentially going this way would be nice, for movies with a thumb shape:

* If there is a thumb, use that (The reasoning being that usually, there won't be one. So if there IS one, the user likely wants to use that specifically)
* If there is no thumb, use the backdrop
* If there is no backdrop, show the icon

I think that, this way, it makes sense and fits all the uses people could be having for the image types we support.